packages/marko/docs/marko-vs-react.md

```diff
@@ -617,20 +617,22 @@ style.less {
 
 ### API
 
-Marko compiles component to JavaScript modules that export an API for rendering
-the component as shown below:
+Marko compiles components to JavaScript modules with exported API for rendering,
+as shown below:
 
 ```js
-require("./components/greeting")
+import greetingComponent from "./components/greeting.marko";
```

Comment:
Maybe just condense to the following?

```js
import Greeting
```

Reply:
Just here, or do you want to move to `PascalCase` for `.marko` imports across the docs from now on?

Reply:
In think either `Template` or the component name in pascal case throughout the docs would be good.

Reply:
I’ll change the instance here to keep this ticket scoped, but we should probably follow up with @mlrawlings to get a Project entry for that particular consistency across the rest of the docs